betatest/templates/betatest/welcome_email.txt

```diff
@@ -0,0 +1,26 @@
+{% autoescape off %}
+Hi {{ application.user.profile.full_name }},
+
+we are pleased to inform you that your application to participate in the
+OpenCraft Instance Manager beta test has been accepted.  Your Open edX instance
+has been provisioned and is available at
```

Comment:
@smarnach I don't think I said this, but for the most part I really liked how the emails were written and I thought you did a great job with them. And yeah, I figured that sort of simple edit issue was what happened here but wanted to be clear what exactly the problem was I was pointing out.